scalafix-tests/input/src/main/scala/test/DisableSyntax.scala

```diff
@@ -57,4 +61,50 @@ case object DisableSyntax {
 
   trait TraitCo[+T]            // assert: DisableSyntax.covariant
   type TypeCo[+T] = TraitCo[T] // assert: DisableSyntax.covariant
+
+  class Foo {
+    def bar(x: Int = 42) = 1 // assert: DisableSyntax.defaultArgs
+  }
+
+  trait FooT {
+    def bar(x: Int = 42) = ???           // assert: DisableSyntax.defaultArgs
+    def foo(a: Int)(b: Int = 1) = ???    // assert: DisableSyntax.defaultArgs
+    def foobar(a: Int, b: Int = 1) = ??? // assert: DisableSyntax.defaultArgs
+  }
+
+  trait FooTT {
+    def bar(x: Int) = 3 // ok
+  }
+
+  val fooT = new FooTT {
+    override def bar(x: Int = 42) = 4 // assert: DisableSyntax.defaultArgs
+  }
+
+  def bar(x: Int = 42) = 5 // ok
+  def foobar = {
+    def foobarfoo(x: Int = 42) = 6 // ok
+  }
+
+  class Buzz(val a: Int = 1) // ok
+
+  abstract class AbstractThing {
+    val noooo = "yes" // assert: DisableSyntax.valInAbstract
+  }
+
+  trait AbstractInterface {
+    val superImportantConst = 42 * 42 * 42 // assert: DisableSyntax.valInAbstract
+  }
+
+  class RealThing extends AbstractThing {
+    val itsok = "indeed" // ok
+  }
+
+  val anon = new AbstractThing {
+    val itsalsook = "indeed" // ok
+  }
+
+  implicit object FooImplicit extends Foo {} // assert: DisableSyntax.implicitObject
+
+  implicit def toString(a: Any): String = a.toString // assert: DisableSyntax.implicitConversion
+  implicit def toImplicitString(implicit foo: Foo) = foo.toString // ok
```

Comment:
I feel like this DisableSyntax.scala test is growing too big, each file should ideally not exceed 50 lines. Small files make it easier to debug breaking tests. Can you please split this into `DisableSyntax2.scala`? (or another name if you can come up with something more creative)